packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -0,0 +1,957 @@
+    await tester.pump();
+
+    final DefaultTextStyle titleStyle = tester.firstWidget(find.widgetWithText(DefaultTextStyle,
+        'Action Sheet'));
+
+    expect(titleStyle.style.fontWeight, FontWeight.w400);
+  });
+
+  testWidgets('Action sheet text styles are correct when message but no title is included',
+          (WidgetTester tester) async {
+    await tester.pumpWidget(
+      createAppWithButtonThatLaunchesActionSheet(
+        const CupertinoActionSheet(
+          message: Text('An action sheet'),
+        ),
+      ),
+    );
+
+    await tester.tap(find.text('Go'));
+    await tester.pump();
+
+    final DefaultTextStyle messageStyle = tester.firstWidget(find.widgetWithText(DefaultTextStyle,
+        'An action sheet'));
+
+    expect(messageStyle.style.fontWeight, FontWeight.w600);
+  });
+
+  testWidgets('Content section but no actions', (WidgetTester tester) async {
+    final ScrollController scrollController = new ScrollController();
+    await tester.pumpWidget(
+      createAppWithButtonThatLaunchesActionSheet(
+        new CupertinoActionSheet(
+              title: const Text('The title'),
+              message: const Text('The message.'),
+              messageScrollController: scrollController,
+            ),
+          ),
+    );
+
+    await tester.tap(find.text('Go'));
+
+    await tester.pump();
+    await tester.pump(const Duration(seconds: 1));
+
+    // Check that there's no button action section.
+    expect(scrollController.offset, 0.0);
+    expect(find.widgetWithText(CupertinoActionSheetAction, 'One'), findsNothing);
+
+    // Check that the dialog size is the same as the content section size (minus padding).
+    expect(
+      tester.getSize(find.byType(ClipRRect)).height,
+      tester.getSize(find.byType(CupertinoActionSheet)).height  - 20.0,
+    );
+
+    expect(
+      tester.getSize(find.byType(ClipRRect)).width,
+      tester.getSize(find.byType(CupertinoActionSheet)).width - 16.0,
+    );
+  });
+
+  testWidgets('Actions but no content section', (WidgetTester tester) async {
+    final ScrollController actionScrollController = new ScrollController();
+    await tester.pumpWidget(
+      createAppWithButtonThatLaunchesActionSheet(
+          new CupertinoActionSheet(
+                actions: <Widget>[
+                  CupertinoActionSheetAction(
+                    child: const Text('One'),
+                    onPressed: () {},
+                  ),
+                  CupertinoActionSheetAction(
+                    child: const Text('Two'),
+                    onPressed: () {},
+                  ),
+                ],
+                actionScrollController: actionScrollController,
+              ),
+        ),
+    );
+
+    await tester.tap(find.text('Go'));
+
+    await tester.pump();
+    await tester.pump(const Duration(seconds: 1));
+
+    // Check that the title/message section is not displayed
+    expect(actionScrollController.offset, 0.0);
```

Comment:
Not sure how the actions scroll controller's offset is indicative of the presence or absence of the title/message section